tck/src/main/resources/reference.conf

```diff
@@ -11,7 +11,7 @@ cloudstate.tck {
     port = ${?TCK_PROXY_PORT}
   }
 
-  frontend {
+  service {
     hostname = "127.0.0.1"
     hostname = ${?TCK_FRONTEND_HOST}
```

Comment:
should `TCK_FRONTEND_HOST` be renamed too to `TCK_SERVICE_HOST `?

Reply:
Yes, forgot about this. I was going to add `TCK_SERVICE_HOST` as well as keep the old `TCK_FRONTEND_HOST` in case it's being used. But I don't see anything that uses these env vars under cloudstate repos, so let's just rename.